devices.json

```diff
@@ -1,6 +1,6 @@
 {
```

Comment:
@EricZielinski This file depends on the specific device used in the experiment, can you update the backbone so that it fetches the path to this file from the config.json file of the experiment?

Reply:
I reverted my changes to one of the device IDs for now.